cypress/integration/rancid_tomatillos_spec1.js

```diff
@@ -0,0 +1,40 @@
+import movieData from './data.js';
+
+describe('App', () => {
+
+  it('should be able to visit app and render the correct elements', () => {
+    cy.visit('http://localhost:3000')
+    const allMovieTitles = movieData.movies.map(movie => movie.title);
+
+    allMovieTitles.forEach(movieTitle => {
+      cy.get('div').contains(movieTitle)
+    });
+
```

Comment:
^ Very clever. Good way to show that ALL the movie titles are displaying on the home page